utilities/transactions/write_prepared_transaction_test.cc

```diff
@@ -55,25 +55,17 @@ TEST(PreparedHeap, BasicsTest) {
   heap.push(34l);
   // Test that old min is still on top
   ASSERT_EQ(14l, heap.top());
-  heap.push(13l);
```

Comment:
With ::AddPrepared being called from the main write queue, the push is now called in order. So we should remove tests with unordered calls to push. This will later help to simplify the implementation of PreparedHeap as well.

Reply:
Is this is only true in case of two write queues though? If so, maybe we should just decide to only support two writes queues.

Reply:
No it is true regardless. I did not mention one write queue because it has always been the case for one write queue to insert into PreparedHeap in order.
But I am in favor of deprecating one-write-queue case, which would make the implementation simpler. We should keep that direction in mind.